news/560.api_change

```diff
@@ -0,0 +1 @@
+OmegaConf.to_yaml now raises a ValueError when called on a str input. Previously an AssertionError was raised.
```

Comment:
There are a handful of user facing function effected by this change (OmegaConf.merge is another). Can you provide a more complete list?

```
OmegaConf.{to_yaml, merge, ...} now raises ...
```

Reply:
I think that it is only `to_yaml / merge / unsafe_merge`.

The OmegaConf.update function also uses `_ensure_contianer` indirectly, but there is an if-statement to make sure that a string is not passed in.